internal/commands/update_stemcell.go

```diff
@@ -12,16 +12,16 @@ import (
 
 	"github.com/pivotal-cf/kiln/internal/commands/flags"
 	"github.com/pivotal-cf/kiln/internal/component"
-	"github.com/pivotal-cf/kiln/pkg/cargo"
 )
 
 type UpdateStemcell struct {
 	Options struct {
 		flags.Standard
 
-		Version        string `short:"v"   long:"version"               required:"true"    description:"desired version of stemcell"`
-		ReleasesDir    string `short:"rd"  long:"releases-directory"    default:"releases" description:"path to a directory to download releases into"`
-		UpdateReleases bool   `short:"ur"  long:"update-releases"                          description:"finds latest matching releases for new stemcell version"`
+		Version         string `short:"v"   long:"version"               required:"true"    description:"desired version of stemcell"`
+		ReleasesDir     string `short:"rd"  long:"releases-directory"    default:"releases" description:"path to a directory to download releases into"`
+		UpdateReleases  bool   `short:"ur"  long:"update-releases"                          description:"finds latest matching releases for new stemcell version"`
+		WithoutDownload bool   `            long:"without-download"                         description:"updates stemcell releases without downloading releases"`
```

Comment:
maybe add a short since all the other options have one? `wd` seems like it would fit the pattern to me